core/common/src/main/java/alluxio/security/authentication/DefaultAuthenticationServer.java

```diff
@@ -56,8 +56,8 @@
   protected final ScheduledExecutorService mScheduler;
 
   /** Interval for clean-up task to fire. */
-  // TODO(gezer) make it configurable.
-  protected final long mCleanupIntervalHour = 1L;
+  protected final long mCleanupIntervalMs =
+      Configuration.getMs(PropertyKey.MASTER_GRPC_SERVER_SHUTDOWN_TIMEOUT);
```

Comment:
This is a very different value, (1 min vs 1 hr), is this intended?

Reply:
Nice catch! Forgot to use the new key.